### PR TITLE
Update Push Notifications section

### PR DESCRIPTION
**Pull request by palpatim**
* Cleanup of existing docs from
  https://docs.aws.amazon.com/aws-mobile/latest/developerguide/add-aws-mobile-analytics.html
* Import docs from https://docs.aws.amazon.com/pinpoint/latest/developerguide/,
  including setup of APNS notifications

Suggested reviewers: @undefobj  @muellerfr

### CODE REVIEW

File: ios/how-to-ios-setup-push-notifications.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
+# Setting Up iOS Push Notifications
+
+Push notifications for iOS apps are sent using Apple Push Notification service (APNs). Before you can send push notifications to iOS devices, you must create an app ID on the Apple Developer portal, and you must create the required certificates.
+
+This section describes how to use the Apple Developer portal to obtain iOS and APNs credentials. These credentials enable you to create an iOS project in AWS Mobile Hub and launch a sample app that can receive push notifications.
+
+You do not need an existing iOS app to complete the steps in this section. After you create an iOS project in Mobile Hub, you can download and launch a working sample app. Mobile Hub automatically provisions the AWS resources that your app requires.
```
Comment (undefobj) [inline ios/how-to-ios-setup-push-notifications.md]: We shouldn't have these instructions as Mobile Hub isn't around any longer. The steps should use `amplify add notifications` like in this guide: https://docs.aws.amazon.com/aws-mobile/latest/developerguide/add-aws-mobile-push-notifications.html

File: ios/push-notifications.md [hunk 2]
```diff
@@ -57,38 +58,44 @@ Use the following steps to connect add push notification backend services to you
...
-                         AWSPinpointConfiguration.defaultPinpointConfiguration(launchOptions: launchOptions))
-                 /** end code copy **/
+            // Other didFinishLaunching code...
 
-                 return true
-             }
-             ```
+            /** start code copy **/
+            // Initialize Pinpoint
+            let pinpointConfiguration = AWSPinpointConfiguration.defaultPinpointConfiguration(launchOptions: launchOptions)
```
Comment (undefobj) [inline ios/push-notifications.md]: Are we using `awsconfiguration.json` here? If so we should show a snippet of the structure and required keys, usually this is under a "Manual Configuration" section. This allows customers to use with an existing appId if they didn't setup using the CLI.

Comment (palpatim) [reply ios/push-notifications.md]: Done. I copied the format of the manual configuration section in https://docs.aws.amazon.com/aws-mobile/latest/developerguide/how-to-integrate-an-existing-bucket.html

File: ios/how-to-ios-setup-push-notifications.md (hunk 1 above)
Comment (palpatim) [reply ios/how-to-ios-setup-push-notifications.md]: Done.

**Comment by undefobj** [comment]
Looks good.

**Pull request merged**